src/test/java/seedu/address/AppParametersTest.java

```diff
@@ -2,7 +2,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
```

Comment:
not an issue at all, but just wondering is there any reason why we're switching from assertTrue to assertNotEquals?

Reply:
Scratch that, I just saw the messages on the group, you can ignore this comment